antlr_ast.py

```diff
@@ -172,7 +174,7 @@ def bind_to_visitor(visitor_cls, node_cls, rule_name, method):
     """Assign AST node class constructors to parse tree visitors."""
     visitor = get_visitor(node_cls, method)
     # TODO raise warning if attr already on visitor?
-    method_name = "visit{}".format(rule_name.capitalize())
+    method_name = "visit{}".format(rule_name[0].upper() + rule_name[1:])
```

Comment:
equivalent?

Reply:
There are names with capitals after the first character in plsql. This was the reason dynamic visitor binding didn't work there.

Reply:
You could make this clear by creating a helper function and write some tests for it to demonstrate desired behaviour.

Reply:
Like https://github.com/datacamp/sheetwhat/blob/master/tests/test_utils.py#L161-L172